demos/js/passthrough.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+(function (fluid) {
```

Comment:
These days I am tending to do without the file-scope closure, but you may still like it - we got rid of all the ones in Infusion, primarily since they used to close over our versioned globals which then made a lot of diff noise on every update.

Reply:
I don't have versioned globals and I prefer the style as long as it's not causing problems.